clear-lake-restoration-app/src/Components/pages/data-archive/CTDData.js

```diff
@@ -114,7 +114,7 @@ function CTDData() {
                 <button className="submitButton" onClick={setGraphDates}>Submit</button>
             </div>
         {profileData.isLoading && <center>Fetching Data...</center>}
-        {!profileData.isLoading && profileData.data.length != 0 && showButton && <CSVLink filename={siteName+"_"+startGraphDate.toISOString().slice(0,10)} data={profilecsv} className="csv-link" target="_blank">Download Profile Data</CSVLink>}
```

Comment:
I think I might have been the one to change this, but could you change the name of the ctd csv file back to ```filename={siteName+"_"+startGraphDate.toISOString().slice(0,10)}```. Since the dates displayed in the data picker are already in UTC.